test/FsAutoComplete.IntegrationTests/TestHelpers.fsx

```diff
@@ -290,14 +273,11 @@ let writeNormalizedOutputWith additionalFn (fn: string) (s: string) =
                                "tmp.*?\.tmp",
                                "<tempfile name removed>")
 
-    // normalize newline char
-    lines.[i] <- lines.[i].Replace("\r", "").Replace(@"\r", "")
+  //   // normalize newline char
+  //   lines.[i] <- lines.[i].Replace("\r", "").Replace(@"\r", "")
 
     lines.[i] <- additionalFn lines.[i]
 
-  //workaround for https://github.com/fsharp/fsharp/issues/774
-  let lines = lines |> Array.filter ((<>) "non-IL or abstract method with non-zero RVA")
```

Comment:
we don't tests on versions of mono that have this problem, so we don't need this anymore.